File: 圆角图片/圆角图片/ViewController.m
//
//  ViewController.m
//  圆角图片
//
//  Created by 侯宝伟 on 2019/5/15.
//  Copyright © 2019 ZHUNJIEE. All rights reserved.
//

#import "ViewController.h"
#import "UIImage+Circle.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    UIImageView *imageView = [[UIImageView alloc] initWithImage:[[UIImage imageNamed:@"noselection"] clipImageWithCornerRadius:60]];
    imageView.frame = CGRectMake(0, 0, 120, 120);
    imageView.center = self.view.center;
    [self.view addSubview:imageView];
}


@end

File: BigImageViewer/BigImageViewer/ViewController.swift
//
//  ViewController.swift
//  BigImageViewer
//
//  Created by zl on 2019/7/30.
//  Copyright © 2019 ZHUNJIEE. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var stybdImage: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let pic = BigImageViewer(image: UIImage(named: "pic"))
        pic.frame = CGRect(x: 100, y: 100, width: 100, height: 100)
        view.addSubview(pic)
    }


}

